extensions/amp-carousel/0.1/scrollable-carousel.js

```diff
@@ -89,6 +89,12 @@ export class AmpScrollableCarousel extends BaseCarousel {
       },
       ActionTrust.LOW
     );
+    /** If the element is in an email document, allow its `goToSlide` action. */
+    Services.actionServiceForDoc(this.element).addToWhitelist(
+      'AMP-CAROUSEL',
+      'goToSlide',
+      {isAllowedInEmail: true}
```

Comment:
Does this need to be an options bag if there's only one option? WDYT about passing a format string instead?

Reply:
Modified to be a list of format strings for scalability considerations described in [separate thread](https://github.com/ampproject/amphtml/pull/27561#discussion_r412629547)